content/doc/book/installing/_kubernetes.adoc

```diff
@@ -0,0 +1,524 @@
+  printf $(kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
+  2. Get the Jenkins URL to visit by running these commands in the same shell:
+  export NODE_PORT=$(kubectl get --namespace jenkins -o jsonpath="{.spec.ports[0].nodePort}" services jenkins)
+  export NODE_IP=$(kubectl get nodes --namespace jenkins -o jsonpath="{.items[0].status.addresses[0].address}")
+  echo http://$NODE_IP:$NODE_PORT/login
+  3. Login with the password from step 1 and the username: admin
+  4. Use Jenkins Configuration as Code by specifying configScripts in your values.yaml file, see documentation: http:///configuration-as-code and examples:  https://github.com/jenkinsci/configuration-as-code-plugin/tree/master/demos
+  +
+  For more information on running Jenkins on Kubernetes, visit:
+  https://cloud.google.com/solutions/jenkins-on-container-engine
+  For more information about Jenkins Configuration as Code, visit:
+  https://jenkins.io/projects/jcasc/
+  ----
+
+. Depending on your environment, it can take a bit of time for Jenkins to start up. Enter the
+  following command to inspect the status of your Pod:
++
+[source,bash]
+----
+kubectl get pods --namespace=jenkins
+----
+
+  Once Jenkins is installed, the status should be  set to Running as in the following output:
++
+[source,bash]
+----
+❯ kubectl get pods --namespace=jenkins
+NAME                       READY   STATUS    RESTARTS   AGE
+jenkins-645fbf58d6-6xfvj   1/1     Running   0          2m
+----
+
+. To access your Jenkins server, you must retrieve the password. You can retrieve your password
+  using either of the two options below.
++
+*Option 1*
++
+Run the following command:
++
+[source,bash]
+----
+printf $(kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}" | base64 --decode);echo
+----
++
+The output should look like this:
++
+[source,bash]
+----
+Um1kJLOWQY
+----
++
+[NOTE]
+====
+👆🏻Note that your password will be different.
+====
++
+*Option 2*
++
+Run the following command:
++
+[source,bash]
+----
+kubectl get secret --namespace jenkins jenkins -o jsonpath="{.data.jenkins-admin-password}"
+----
++
+The output should be a base64 encoded string like this:
++
+[source,bash]
+----
+WkIwRkdnbDZYZg==
+----
++
+Decode the base64 string and you have your password. You can use this website to decode your output. 
+
+. Get the name of the Pod running that is running Jenkins using the following command:
++
+[source,bash]
+----
+Kubectl get pods --namespace jenkins
+----
+
+. Use the kubectl command to set up port forwarding:
++
+[source,bash]
+----
+❯ kubectl --namespace jenkins port-forward <pod_name> 8080:8080
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+----
+
+Visit http://127.0.0.1:8080/ and log in using `admin` as the username and the password you retrieved earlier.
+
+==== Installing Jenkins on Kubernetes without Helm or operators
```

Comment:
May want to add a section above this as a placeholder for Operators with "coming soon" or something like that.

Reply:
This is in the second pull request